_posts/2022-01-10-reassign-partitions.md

```diff
@@ -0,0 +1,374 @@
+---
+layout: post
+title:  "Reassigning partitions in Apache Kafka Cluster"
+date: 2021-09-23
+author: shubham_rawat
+---
+
+As Apache Kafka users, we commonly deal with brokers and the partitions. 
+Sometimes you might want to move the partitions from old brokers to the new brokers or maybe you want to change the ordering of the partition assignment list.
```

Comment:
The old broker and new broker reference seems weird as Strimzi has no such concept currently. So I think it can be confusing. So maybe it can be _from one broker to another_?